ImageItem/ImageItem.module.less

```diff
@@ -11,7 +11,15 @@
 		}
   	}
 
-  	.applySkins({
+	&.captionOverImage {
+		.caption {
+			overflow: hidden;
+			position: absolute;
+			bottom: 6px;
```

Comment:
maybe this could be a variable, only in variables-base. it will be easier later to customize this for other skins